README.md

```diff
@@ -197,13 +197,13 @@ Usage docs can be found here: [Usage](Usage.md)
 Because TestNG is built into the framework, the functionality available there is applicable for annotating tests.  More information is available here: http://testng.org/doc/
 
 ### Command Line
-The built-in functionality of Maven/failsafe in addition to the aspects built into Ride, allows for simple execution of run tests as outlined below (from within the test repo root):
+The built-in functionality of Maven/failsafe in addition to the aspects built into Ride allows for simple execution of run tests as outlined below (from within the test repo root):
```

Comment:
For this one, I think it reads better if you don't remove the comma, but add another comma after "failsafe"

Reply:
Did it in the next commit as you told.